docs/project-backlog.md

```diff
@@ -0,0 +1,122 @@
+
+* **Reviewer**: Person(s) whose Approval is needed to merge the PR.
+
+* **Contributing Reviewer**: Person who provides feedback on an PR but whose Approval is not necessarily needed to merge the PR.
+
+## SLAs
+We value having a responsive backlog, and have response SLAs for each role. If an issue exceeds an SLA, it is put into the `stale` state.
+
+* **Triager**: Daily (Business Days) review of new issues with initial feedback to OP or issue processed within 3 days of last activity.
+* **Reviewer / Contributing Reviewer**: Provide feedback within 3 days of the last activity on a PR
+* **Contributor**: 3 day updates as comments on the PR to indicate activity
+
+## Issue States
+
+### New
+label: `new`
+
+New issues are the domain of the backlog Triagers. Every new issue should be responded to quickly, with a greeting and any needed clarifying questions. If assessing the issue requires more information, the Triager collaborates with the OP until workability is determined.
+
+The most common next steps for new tickets:
+* `available` for actionable work items.
+* `needs-discussion` for issues which cannot be triaged quickly, but raise important points.
+* `no-further-action` for issues which can be quickly resolved.
+
+### Available
+label: `available`
+
+For new work requests Once it has been classified, a ticket is up for grabs to be picked up to be worked on by a contributor.
+
+All work is labeled with a priority:
+* `P0` = Critical (Drop everything and work on it.)
+* `P1` = Important (Work on it if no P0 are present. Use judgment to prioritize items within a list of P1s)
+* `P2` = Low (Work on it when no P1 are left or some one from community wants to contribute)
+
+Work can also be classified into a number of categories, to make available issues easier to find. Common examples of work categories are:
+* `feature-request`
+* `enhancement`
+* `bug`
+* `docs`
+
+Available work can be assigned in the following ways:
+* A contributor assigns the ticket to themselves.
+* A maintainer assigns the ticket to a contributor.
+
+All tickets transition to `wip` once they have been assigned.
+
+### WIP
+label: `wip`
+
+Any ticket which has been assigned a lead contributor is considered work in progress. 
+
+* If a PR is merged which resolves the issue, the ticket can be closed as `completed`.
+* If work is blocked on review, the ticket transitions to `needs-review`.
+* If work is abandoned, or the ocntributors otherwise change their minds, the ticket can be closed as `no-further-action`.
+
+### Needs Review
+label: `needs-review`
+
+Implementation work related to the ticket has been completed. Work related to a Review and/or any changes requested as part of the review  is in progress.  Issues with a PR associated with it will be considered as in ‘In Review’.
```

Comment:
same question - "considered `In Review`" - is there manual step for this? If so - it sounds heavyweight process. If there is an automation - how it will understand issue to PR association?

Reply:
I feel this is related to SLAs. Ideally, no state change from “wip” is needed. However, if an issue becomes blocked on review activity, it’s state should change. 

I would like automation to track SLAs, and assign labels in response. But the task should be simple enough for a triager to manually.

Reply:
so you want to solve the problem that the issue become "stale" in `wip` state? If there is an automation that will bug me to change the state - I will do it. Otherwise - it looks like too much to do to fix a simple bug (for instance).